stacks/root.yml

```diff
@@ -900,6 +907,48 @@ Resources:
           Value: !Ref AWS::StackId
       TemplateURL: !Join ["", ["http://s3", !If [IsUsEast1, "", "-"], !If [IsUsEast1, "", !Ref "AWS::Region"], ".amazonaws.com/", "Fn::ImportValue": !Sub "${InfrastructureStorageStackName}-InfrastructureSourceBucket", "/", !Ref InfrastructureGitCommit, "/stacks/proxy.prx.org.yml"]]
       TimeoutInMinutes: 5
+  StyleguideStack:
+    Type: "AWS::CloudFormation::Stack"
+    Condition: "IsStaging"
```

Comment:
I happened to see this in the PR to add Metrics, and it seemed like a good idea, so I swiped it.

Reply:
+1.  Staging-only should work fine for styleguide.  Save some ECS space.